node-operators/networks/collator.md

```diff
@@ -33,19 +33,21 @@ From a technical perspective, collators must meet the following requirements:
 
 ## Accounts and Staking Requirements {: #accounts-and-staking-requirements } 
 
-Similar to Polkadot validators, you need to create an account. For Moonbeam, this is an H160 account or an Ethereum-style account from which you hold the private keys. In addition, you will need a minimum amount of tokens staked to be considered eligible (become a candidate). Only a certain number of the top collator candidates by delegated stake will be in the active set of collators.
+Similar to Polkadot validators, you need to create an account. For Moonbeam, this is an H160 account or an Ethereum-style account from which you hold the private keys. In addition, you will need a minimum amount of tokens staked (self-bonded) to be considered eligible and become a candidate. Only a certain number of the top collator candidates by delegated stake, including self-bonded and delegator-bonded stake (total bonded), will be in the active set of collators.
```

Comment:
I would revise "the top collator candidates by delegated stake" to _the top collator candidates by total stake_